### ISSUE TITLE
Jaws does not read first column header in a grid

### ISSUE THREAD

**Issue by Wildebrew** [FreedomScientific/VFO-standards-support#940]
When navigating a grid with arrow keys, Jaws announces all associated column headers except for the first column.

As an example, browse on over to [this APG grid example](https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/)
1. Press tab key until you enter the grid, focus lands on the first date in the date column (Jas does not announce the column header, only the value of the date cell).
2. Now navigate away with arrow right, you hear the values of the cells in a row + the column headers, as expected (type, description, etc.).
3. Navigate back with arrow left. You hear all column headers as expected until you get to the date column, then you do not hear the colummn header, just the value of the cell.

I've seen this consistently happen in other grids, such as AG Grid examples.

Browser/verson:
Latest version of Chrome + Jaws 2026 (602.49).

**Comment by JeremiahRogersVispero** [comment]
@Wildebrew , can you please check with J2026.2603.86 and let me know if the issue persists for you? I cannot reproduce.

**Comment by JeremiahRogersVispero** [comment]
Unable to reproduce with JAWS 2026.2603.86.
